app/controllers/bookings_controller.rb

```diff
@@ -1,46 +1,52 @@
 class BookingsController < ApplicationController
-  before_action :find_dress, only: %i[new show create destroy]
+  before_action :find_dress, only: %i[new create destroy]
 
-  # def index
-  #   # @bookings = policy_scope(Booking).order(created_at: :desc)
-  #   @bookings = Booking.all
-  # end
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
+  end
 
   def new
     @booking = Booking.new
+    authorize @booking
   end
 
-  # def show
-  #   @booking = Booking.find(params[id])
-  # end
+  def show
+    @booking = Booking.find(params[:id])
+    @dress = @dress = Dress.find(@booking.dress_id)
```

Comment:
```suggestion
    @dress = @booking.dress
```
Since you already have the Booking and you have the relation on the model already. 

( I am Dillon Wyatt from the Le-Wagon slack. Future TA )

Reply:
Thank you! That makes a lot of sense ^_^